website/docs/docs/build/udfs.md

```diff
@@ -0,0 +1,208 @@
+---
+title: "User-defined functions"
+description: "Learn how to add user-defined functions (UDFs) to your dbt projects."
+id: "udfs"
+---
+
+# User-defined functions <Lifecycle status="beta" />
```

Comment:
must be on `1.11` or `latest` or `fusion` - do we want to call that out anywhere?